pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1482,17 +1493,50 @@ func (og *operationGenerator) verifyVolumeIsSafeToDetach(
 			"DetachVolume failed fetching node from API server",
 			fmt.Errorf("node object retrieved from API server is nil"))
 	}
+	if fetchErr != nil {
+		if errors.IsNotFound(fetchErr) {
+			klog.Warningf(volumeToDetach.GenerateMsgDetailed("Node not found on API server. DetachVolume will skip safe to detach check", ""))
+		} else {
```

Comment:
This code does not maintain same logic for skipping verification check. It is returning with error *before* verifying if error is `NotFound` error.  This code is not skipping verification check if node is not found.